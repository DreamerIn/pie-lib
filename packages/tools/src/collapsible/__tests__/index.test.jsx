import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import { Collapsible } from '../index';
import React from 'react';

describe('protractor', () => {
  describe('snapshot', () => {
    it('renders', () => {
      const wrapper = shallow(<Collapsible classes={{}} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
