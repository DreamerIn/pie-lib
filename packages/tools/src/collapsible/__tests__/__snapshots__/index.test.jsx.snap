// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`protractor snapshot renders 1`] = `
<div>
  <div
    key="rationale-show-more"
    onClick={[Function]}
  >
    <span>
      Show
    </span>
  </div>
  <WithStyles(Collapse)
    in={false}
    timeout="auto"
    unmountOnExit={true}
  >
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": undefined,
        }
      }
    />
  </WithStyles(Collapse)>
</div>
`;
